### ISSUE TITLE
GeoJSON Support

### ISSUE THREAD

**Issue by jameswm2** [TissUUmaps/TissUUmapsCore#8]
Hi all,

Is there currently a way to convert GeoJSON (https://geojson.org/) polygons to those types that are supported by TissUUmaps?

I am not exactly sure what the input formatting is for your polygon csv type

**Comment by arundasan91** [comment]
Hi @cavenel , any updates on this?

**Comment by cavenel** [comment]
Hi, yes the last version in the [develop](https://github.com/wahlby-lab/TissUUmaps/tree/develop) branch now supports GeoJSON directly. So you can load GeoJSON files exported from QuPath and co, or old json format from TissUUmaps.
This branch should be merged to master soon, and is already the one used in the standalone version (see https://tissuumaps.github.io/download/ for details about installation)

I will wait until develop is merged into master and some feedback before I close this issue.

**Comment by jameswm2** [comment]
@cavenel 
Hi, that is excellent! I just downloaded the standalone version, I tried to import cellular-level segmentation polygons (in my case ~60,000 polygons), but I found that it does not work. Is this an issue on my end, or is the purpose of your geoJSON implementation more for tissue level ROIs?

**Comment by cavenel** [comment]
Yes, we have a better support for tissue level ROIs (for now). Supporting 60K polygons would require some changes in the way the polygons are drawn unfortunately.
Could you share your GeoJSON file so that I can test it and see the main issues?

**Comment by jameswm2** [comment]
Yes, here you go. It is the output for a large ISS segmentation done using Baysor. 
Megaupload link here: https://mega.nz/file/yTJV0YqK#2TeAagUcofIBN5oUHnynC7A276cdNzUstiXW3SL8hiY

**Comment by cavenel** [comment]
Thanks, I was able to load it by modifying a bit the TissUUmaps code (the GeometryCollection type of GeoJSON was not supported) but it contains 124714 regions, so it takes forever to display and makes it completely unusable. What would be the use case? For the future we might want to move the region display to WebGL, as we did with markers.
![image](https://user-images.githubusercontent.com/6164758/142726266-5afe3c7b-dbfb-490a-bc55-10d89c203383.png)
